cms/envs/common.py

```diff
@@ -916,6 +916,9 @@
     # django-oauth-toolkit
     'oauth2_provider',
 
+    # django-oauth-plus
+    'oauth_provider',
```

Comment:
@efischer19 sorry it was something related to our own instance. I remove it from this PR.